<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PerformLogin</class>
 <widget class="QWidget" name="PerformLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Logging in to Twitch ...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QProgressBar" name="prgStep">
     <property name="value">
      <number>0</number>
     </property>
     <property name="format">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stkFeedback">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pagNoToken">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lblTokenExplain">
         <property name="text">
          <string>Qtwitch currently has no Twitch Authentication Token for this profile. You need to authorize this application to control your stream details. When you press the button below, your internet browser will open to a small website that will explain the procedure to you.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stkTokenFallback">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="pagTokenDefault">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="btnAcquire">
             <property name="text">
              <string>Acquire token!</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pagTokenFallback">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="lblTokenFallback">
             <property name="styleSheet">
              <string notr="true">color: red;</string>
             </property>
             <property name="text">
              <string>There was an error opening your browser. Instead, the link has been copied to your clipboard. Please open your internet browser and go to the following link:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="txtTokenFallback">
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
             <property name="margin">
              <number>5</number>
             </property>
             <property name="textInteractionFlags">
              <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lblToken">
           <property name="text">
            <string>Token:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtToken">
           <property name="placeholderText">
            <string>&lt;insert token code here&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnToken">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Proceed</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>txtToken</sender>
   <signal>returnPressed()</signal>
   <receiver>btnToken</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>542</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
